### PR TITLE
Add notes for SIMD meeting 2021-12-03

### PR DESCRIPTION
**Pull request by penzn**

### CODE REVIEW

**Comment by penzn** [comment]
@dtig @akirilov-arm @abrown please feel free to point out if I missed anything

File: simd/2021/SIMD-12-03.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- Various forms of blending to put the result back together
+
+The problem is that mitigations for the constraints above interact with each
+other, making overall sequence more complicated. Removing even one of the
+constraints will make the sequence much simpler. Using more registers may help
+as well.
+
+With AVX512:
+
+- `vrange` has the desired behavior v.r.t signed zeros
```
Comment (abrown) [inline simd/2021/SIMD-12-03.md:36]:
```suggestion
- `vrange` has the desired behavior w.r.t signed zeros
```

**Review by abrown — APPROVED** [review]

File: simd/2021/SIMD-12-03.md (hunk 1 above)
Comment (penzn) [reply simd/2021/SIMD-12-03.md:36]: Good catch!

**Review by dtig — APPROVED** [review]

**Comment by penzn** [comment]
Decided to squash and make commit message a bit more informative.

**Comment by penzn** [comment]
Looks like I don't have write access to this repo. @dtig, @dschuff are subgroup chairs supposed to have write access (or other "contributor" membership) to `meetings`?

**Comment by dtig** [comment]
Following up over email about permissions, merging this PR for now.

**Pull request merged**